### PR TITLE
[Microsoft 365 Lighthouse] - Audit events update

### PR DESCRIPTION
**Pull request by idwilliams**
Updates to address #16132

### CODE REVIEW

File: api-reference/beta/api/managedtenants-auditevent-get.md [hunk 1]
```diff
@@ -30,8 +30,7 @@ One of the following permissions is required to call this API. To learn more, in
 }
 -->
 ``` http
-GET /auditEvent
-GET /managedTenant/auditEvents/{auditEventId}
+GET /tenantRelationships/managedTenants/auditEvents/{auditEventId}
 ```
```
Comment (isvargasmsft) [inline api-reference/beta/api/managedtenants-auditevent-get.md:34]: Checking at the issue, the correction was due in line 62. Could you please clarify the change in the HTTP request?

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Please clarify the HTTP request change. According to the issue the correction was due in the example.

**Comment by isvargasmsft** [comment]
Hello @idwilliams, 

What's the status of this PR? Please let me know if you have questions about my comments above.

Here to help!

**Comment by lramosvea** [comment]
@idwilliams just a heads up on this PR so we can close https://github.com/microsoftgraph/microsoft-graph-docs/issues/16132. Thanks.

**Comment by idwilliams** [comment]
@isvargasmsft the HTTP request was updated because the incorrect URI was used when the article was first published. All functionality for Microsoft 365 Lighthouse, should be prefixed with `/tenantRelationships/managedTenants`. So, this change is correct the URI to match what is available through the schema. Please let me know if you have any concerns or questions.

File: api-reference/beta/api/managedtenants-auditevent-get.md (hunk 1 above)
Comment (idwilliams) [reply api-reference/beta/api/managedtenants-auditevent-get.md:34]: Apologizes for the delay on this, the address in the documentation was incorrect. This update is to have it match what is available through the schema. Below I shared an additional comment with more context.

**Comment by isvargasmsft** [comment]
> @isvargasmsft the HTTP request was updated because the incorrect URI was used when the article was first published. All functionality for Microsoft 365 Lighthouse, should be prefixed with `/tenantRelationships/managedTenants`. So, this change is correct the URI to match what is available through the schema. Please let me know if you have any concerns or questions.
> 
> To my knowledge the samples are automatically generated by tooling the team has in place. So, I did not update the sample only the HTTP request because as I understand it the sample will be corrected next time the tooling updates the samples.

Thank you very much for the clarification, @idwilliams. Regarding the examples, the tool generates the code snippets using the http URL in the example. Please update the http code blocks in the examples with the correct URL.

**Review by isvargasmsft — APPROVED** [review]

**Pull request merged**